artemis-journal/src/main/java/io/netty/buffer/UnpooledUnsafeDirectByteBufWrapper.java

```diff
@@ -563,7 +564,11 @@ protected SwappedByteBuf newSwappedByteBuf() {
    @Override
    public ByteBuf setZero(int index, int length) {
       if (hasMemoryAddress()) {
-         UnsafeByteBufUtil.setZero(this, addr(index), index, length);
+         if (length == 0) {
```

Comment:
We have to do this as UnsafeByteBufUtil.setZero(this, addr(index), index, length); util method is removed in 4.1.14 this was the underlying util logic.